lib/internal/abort_controller.js

```diff
@@ -137,8 +138,35 @@ function setWeakAbortSignalTimeout(weakRef, delay) {
 }
 
 class AbortSignal extends EventTarget {
-  constructor() {
-    throw new ERR_ILLEGAL_CONSTRUCTOR();
+
+  /**
+   * @param {symbol | undefined} dontThrowSymbol
+   * @param {{
+   *   aborted? : boolean,
+   *   reason? : any,
+   *   transferable? : boolean,
+   *   composite? : boolean,
+   * }} [init]
+   * @private
+   */
+  constructor(dontThrowSymbol = undefined, init = kEmptyObject) {
```

Comment:
It is breaking the DOM Standard?

Reply:
to explain a little bit: `AbortSignal.constructor.length` should be 1, the `= undefined` should be removed here.

Reply:
If it break the spec, shouldn't web platform tests fail?

Reply:
I don't think it ever test the constructor argument.
https://github.com/web-platform-tests/wpt/blob/b72282f5f257b51212cfbc9b0fb7fbbd8e982583/dom/abort/AbortSignal.any.js#L15

But one never expect the arguments to change without the specification.

Reply:
`AbortSignal.length` is already `0` in the current version of Node.js, so this PR doesn't change it.

Reply:
I see it's also `0` in Chrome.
`AbortSignal.constructor.length` is `1`. It's the length of the `Function` constructor.

Reply:
@climba03003 given @targos comment is your comment a blocker?

Reply:
Personally, still doesn't see it as spec compliance.

It maybe acceptable because there still no way for the user to create `AbortSignal` by `new AbortSignal()`.

Reply:
Will create a different pr for making it spec compliant